pkg/marketinfo/marketinfo.go

```diff
@@ -0,0 +1,41 @@
+package marketinfo
+
+import (
+	"time"
+
+	"github.com/tdex-network/tdex-feeder/config"
+)
+
+// MarketInfo stores the informations necessary for
+// handling different market pair prices in real-time.
+type MarketInfo struct {
+	config config.Market
```

Comment:
if you make those files exported (ie captial letter) you could drop getter and setters, this ain't Java